README.md

```diff
@@ -15,7 +15,7 @@ Example
 steps:
   - command: bin/ci-aws-thing
     plugins:
-      - cultureamp/aws-assume-role#v0.1.0:
+      - envato/aws-assume-role#v0.2.0-envato:
```

Comment:
Why the need to specify `-envato` to the version if it's already using the `envato/aws-assume-role` repo?

Reply:
Given this is fork, of `cultureamp/aws-assume-role`. I don't want the tag to clash with a potential `0.2.0` released upstream.

Reply:
Doesn't this PR bring this fork up to date _and identical_ to the parent repo? If so, there shouldn't be a difference (unless I'm missing something?)

Reply:
No, the change was reverted in the parent repo: https://github.com/cultureamp/aws-assume-role-buildkite-plugin/pull/11